packages/firestore/src/lite-api/aggregate_types.ts

```diff
@@ -71,19 +71,23 @@ export type AggregateSpecData<T extends AggregateSpec> = {
 /**
  * The results of executing an aggregation query.
  */
-export class AggregateQuerySnapshot<T extends AggregateSpec> {
+export class AggregateQuerySnapshot<
+  AggregateSpecType extends AggregateSpec,
+  AppModelType = DocumentData,
+  DbModelType extends DocumentData = DocumentData
```

Comment:
I get why we added these generic params to the AggregateQuerySnapshot class, but I'm concerned if they justify the complexity. The general use case of getting data from an aggregate query snapshot isn't concerned with the app model and db model types. Only in the case where someone is referencing the query object from the snapshot do those types become useful. The only drawback is the added complexity of the documentation, and the risk that this is a point of confusion for readers.

Reply:
Discussed above.